src/lxc/utils.h

```diff
@@ -620,7 +620,7 @@ extern int lxc_set_death_signal(int signal);
 extern int fd_cloexec(int fd, bool cloexec);
 extern int recursive_destroy(char *dirname);
 
-#define lxc_split_string(__iterator, __splitme, __separators)                   \
+#define lxc_iterate_parts(__iterator, __splitme, __separators)                  \
```

Comment:
That should be in the previous commit :)

Reply:
Oh weird, sorry. :)